docs/guides/protect-page-login/php.mdx

```diff
@@ -0,0 +1,135 @@
+---
+id: php
+title: Protect a Page with Login in PHP
+sidebar_label: PHP
+---
+
+```mdx-code-block
+import Teaser from './_common/teaser.mdx'
+import mp4 from './php/screencast.mp4'
+import webm from './php/screencast.webm'
+
+<Teaser
+  framework={<a href="https://www.php.net/">PHP</a>}
+  mp4={mp4}
+  webm={webm}
+/>
+```
+
+## Create PHP App
+
+First we create a new PHP project:
+
+```shell-session
+mkdir your-project
+cd your-project
+touch index.php
+```
+
+## Install Dependencies
+
+To interact with Ory's APIs we install the [Ory SDK](https://packagist.org/packages/ory/client):
+
+```shell-session
+composer require ory/client:^v0.0.1-alpha
+```
+
+To simplify URLs handling we install the [bramus/router](https://packagist.org/packages/bramus/router)
+```shell-session
+composer require bramus/router
+```
+
+### Why do I Need the Ory CLI?
```

Comment:
Is this shared between all guides? Can we make it an importable component then, so that we only have to fix it in one place later on?

Reply:
good idea, I'll move this section to the `_common` dir